spec/models/source_spec.rb

```diff
@@ -5,7 +5,7 @@
   it { is_expected.to validate_presence_of(:url) }
 
   describe '.for_accepted_teams' do
-    let!(:accepted_team)   { create :team, kind: %w(sponsored voluntary).sample }
+    let!(:accepted_team)   { create :team, kind: %w(sponsored deprecated_voluntary).sample }
```

Comment:
Same here: please remove the deprecated columns and test for the new things instead.